charts/kyverno/values.yaml

```diff
@@ -343,13 +350,12 @@ config:
   # The Kyverno namespace is excluded if `excludeKyvernoNamespace` is `true` (default)
   webhooks:
     # Exclude namespaces
-    # - namespaceSelector:
-    #     matchExpressions:
-    #     - key: kubernetes.io/metadata.name
-    #       operator: NotIn
-    #       values:
-    #         - kube-system
-    #         - kyverno
+    - namespaceSelector:
+        matchExpressions:
+        - key: kubernetes.io/metadata.name
+          operator: NotIn
+          values:
+            - '{{ include "kyverno.namespace" . }}'
```

Comment:
Do we want this ? We already have the `excludeKyvernoNamespace` for that.

Reply:
I had made the change as was asked to make the values dynamic. Can remove this if not required.

Reply:
I think `excludeKyvernoNamespace` is preferable because we know what the user wants to do and implement the logic.